web-client/src/models/requests/RequestWithOffersAndTimeline.ts

```diff
@@ -100,29 +101,37 @@ export class RequestWithOffersAndTimeline extends Request
   }
 
   @IsArray()
-  private _timeline: TimelineItem[];
+  private _timelineItems: TimelineItem[];
 
   get timeline(): TimelineItem[] {
-    return this._timeline;
+    return this._timelineItems;
   }
 
   set timeline(timelineItems: TimelineItem[]) {
-    this._timeline = timelineItems;
+    this._timelineItems = timelineItems;
   }
 
   public addToTimeline(timelineItem: TimelineItem) {
-    this._timeline.push(timelineItem);
+    this._timelineItems.push(timelineItem);
   }
 
   public getRequest(): Request {
     return Request.factory(this.toObject() as IRequest);
   }
 
   public static factory(
-    data: IRequestWithOffersAndTimeline,
+    data: IRequestWithOffersAndTimelineItems,
   ): RequestWithOffersAndTimeline {
+    console.log('z3', data.pinUserRef);
+    console.log('z3a', data.requestId);
+    console.log('z3b', data.requestRef);
+    console.log('z3c', data.timeline);
     return new RequestWithOffersAndTimeline(
```

Comment:
The below code I commented out gave an error.  The code that is commented out also does not look correct as db.doc takes a string and data.pinUserRef worked.